ExternalModule.php

```diff
@@ -25,15 +25,25 @@ function hook_survey_complete($project_id, $record, $instrument, $event_id, $gro
 
       global $Proj, $redcap_version;
 
+      // Get values from settings.
       $source_instruments = AbstractExternalModule::getProjectSetting('ssptf_source_instrument');
+      $active_instruments = AbstractExternalModule::getProjectSetting('ssptf_target_active_field');
+      $compact_pdf = AbstractExternalModule::getProjectSetting('ssptf_compact_pdf');
+      
+      // Safety check to see to make sure compact_pdf has a value as not manditory  
+      if (!$compact_pdf)
+                $compact_pdf = 'false';
```

Comment:
'false' is a string.. which will evaluate to true.. https://www.php.net/manual/en/language.types.boolean.php
if your goal is a default of false then set it to _FALSE_